doc/nrf/protocols/matter/getting_started/hw_requirements.rst

```diff
@@ -57,37 +57,45 @@ Values are provided in kilobytes (KB).
+      | :ref:`Light Switch <matter_light_switch_sample>` (Debug)           |            28 |               757 |              4 |         32 |         821 |                             176 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Light Switch <matter_light_switch_sample>` (Lto debug)       |            28 |               693 |              4 |         32 |         757 |                             180 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Light Switch <matter_light_switch_sample>` (Release)         |            28 |               654 |              4 |         32 |         718 |                             169 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Lock <matter_lock_sample>` (Debug)                           |            28 |               770 |              4 |         32 |         834 |                             177 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Lock <matter_lock_sample>` (Lto debug)                       |            28 |               705 |              4 |         32 |         769 |                             181 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Lock <matter_lock_sample>` (Release)                         |            28 |               654 |              4 |         32 |         718 |                             172 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Template <matter_template_sample>` (Debug)                   |            28 |               710 |              4 |         32 |         774 |                             174 |
+      +--------------------------------------------------------------------+---------------+-------------------+----------------+------------+-------------+---------------------------------+
+      | :ref:`Template <matter_template_sample>` (Lto debug)               |            28 |               653 |              4 |         32 |         717 |                             178 |
```

Comment:
```suggestion
      | :ref:`Template <matter_template_sample>` (LTO Debug)               |            28 |               653 |              4 |         32 |         717 |                             178 |
```
like before. Also:
- has template LTO Release variant been dropped intentionally? I guess it was meant to demonstrate the very minimum usage for Matter.
- is adding LTO Debug variant for all samples intentional?

Reply:
There is an issue in our script and LTO variant is not generated properly, I'm investigating it. The LTO release variant was not dropped intentionally, memory requirements table updating is still in progress. Regarding Debug LTO variants, yes it was intentional to have it, but we can discuss it f2f. :)

Reply:
ok, I was just double checking, but if it was intentional then that's fine with me :)

Reply:
Regarding the LTO release variant, we decided to not include it in the 2.7.0 release scope, but we're planning to add it once the LTO becomes no experimental. For now, we go with LTO + debug only.